### DISCUSSION TITLE
Licence of AdwCustomizer

### DISCUSSION THREAD

**Discussion by daudix** [GradienceTeam/Gradience#87] [General]
New design repo have GPLv3 license, most of other repos use GPLv3, but main repo uses MIT, this can make an issues in the future, from integrating symbolic icons to code blocks, we need to make somthing about this.

**Comment by 0xMRTT** [comment]
Otherwise we can change the license of all others repo to MIT.

**Reply by daudix** [reply]
Is there any advantages over GPLv3?

**Reply by 0xMRTT** [reply]
https://choosealicense.com/licenses/mit/

**Reply by 0xMRTT** [reply]
https://choosealicense.com/licenses/gpl-3.0/. Personally, i prefer GNU GPLv3

**Comment by ArtyIF** [comment]
You have to change the license in the files too, pretty much all files lines 5-27 still have the MIT license.

**Reply by ArtyIF** [reply]
Other GPL projects use this:

```
This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.
 
You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
